client/js/libs/handlebars/index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+module.exports = {
+	colorClass: require("./colorClass"),
+	diff: require("./diff"),
+	equal: require("./equal"),
+	firstLetterUppercase: require("./firstLetterUppercase"),
+	localedate: require("./localedate"),
+	localetime: require("./localetime"),
+	modes: require("./modes"),
+	parse: require("./parse"),
+	roundBadgeNumber: require("./roundBadgeNumber"),
+	slugify: require("./slugify"),
+	ternary: require("./ternary"),
+	tojson: require("./tojson"),
+	tz: require("./tz"),
+	users: require("./users"),
+};
```

Comment:
Was that unnecessary before due to the Webpack/Handlebars integration?
Does this make [that](https://github.com/thelounge/lounge/blob/8658f15751606da72809dd02545a36b4cf7813bc/webpack.config.js#L52-L54) unnecessary?